index.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
+var path = require('path');
 var timers = require('timers');
 var process = require('process');
 var assert = require('assert');
-var _gcstats;
+var EventEmitter = require('events').EventEmitter;
 var _toobusy;
 
+var _gcEmitter = new EventEmitter();
+/* eslint-disable camelcase */
+var _gcstats = require('bindings')({
```

Comment:
Sorry; not here. Please do a delayed side effect in the constructor very similar to the delayed toobusy side effect.